fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -507,6 +507,9 @@ public class OperationType {
     // alter catalog
     public static final short OP_ALTER_CATALOG = 13300;
 
+    // alter datacache info of lake table
+    public static final short OP_ALTER_DATACACHE_PARTITION_DURATION = 13400;
```

Comment:
there is no additional processing for data cache partition duration, so i think there is no need to add a new operation type, and use `OP_ALTER_TABLE_PROPERTIES` is fine.